config/scheduled_jobs.yml

```diff
@@ -6,3 +6,7 @@
 #   cron: "* * * * * *" # also accepts natural language, e.g. "every second"
 #   class: NameOfAJob
 #   description: Description is shown in the GoodJob dashboard.
+trn_mentor_sync:
+  cron: "0 0 * * *"
+  class: "TRSMentorSyncJob"
+  description: "Sync all mentor first and last name attributes"
```

Comment:
I feel like these naming conventions might be slightly awkward.

The scheduled job is called **trn**_mentor_sync but the job class is **TRS**MentorSyncJob.

I wonder if we need to use these acronyms at all? 🤔 Is it important that TRN/TRS forms part of the name of the job? Within the app, we refer to these simply as "mentors".

What about if we called our jobs:

- `SyncAllMentorsJob`
- `SyncMentorJob`

or something?